src/cmdstan-guide/print.Rmd

```diff
@@ -1,3 +1,7 @@
+---
+pagetitle: Deprecations
+---
+
```

Comment:
Maybe the wrong title?

Reply:
fixed - I think that there might have been a `Deprecated` section that got lost.